apps/web-giddh/src/app/settings/financial-year/financial-year.component.ts

```diff
@@ -88,8 +88,8 @@ export class FinancialYearComponent implements OnInit {
             if (o) {
                 this.financialYearObj = _.cloneDeep(o);
                 let yearOptions = _.cloneDeep(this.yearOptions);
-                o.financialYears.forEach((fYear) => {
-                    let year = moment(fYear.financialYearStarts, GIDDH_DATE_FORMAT).year();
+                o.financialYears.forEach((fyear) => {
```

Comment:
Make sure these changes not effecting on funcationality

Reply:
done